src/components/elements/FilterBox.js

```diff
@@ -0,0 +1,53 @@
+import { useRef } from 'react';
+import classes from "./FilterBox.module.css";
+
+function FilterBox(props) {
```

Comment:
Have tests for this addition?

Reply:
Would the tests be included here in the React side then? Haven't done testing in JS before - any particular libraries?

Reply:
I forgot you weren't doing tests for this project!

In consideration for time I'll have to let this slip.

In a real project, all code needs to be tested. The React side will get all the usual benefits of having tests here.